### ISSUE TITLE
WASD for creative flying

### ISSUE THREAD

**Issue by LonelyFear** [OfficialCodeNoodles/2D-Minecraft#24]
I feel like using the shift key for going down is a bit, clunky. I think that the W and S keys can be used for flight instead, for up and down respectively. This could also extent to sneaking and jumping in survival too.

**Comment by MadCatterDev** [comment]
I agree with the creative flight part, but I think that in survival, W and S should be used for going up and down ladders after those are added.
